### ISSUE TITLE
Height not properly calculated in IE 11 if using font-size: inherit

### ISSUE THREAD

**Issue by cecton** [andreypopp/react-textarea-autosize#198]
## Demo 

https://codesandbox.io/s/20lwok4oor

## Browsers

Works correctly on Edge, Chrome, FireFox but not on IE.

## Problem Description

The textarea height is miscalculated, it uses the font-size of the document instead of the div element.

Relevant css:

```css
textarea {
  font-size: 13px;
}

.container {
  font-size: 24px;
}

.container > textarea {
  font-size: inherit;
  width: 100%;
  overflow: hidden;
  resize: none;
  border: 0;
  outline: 0;
  background-color: lightblue;
}
```

**Comment by Andarist** [comment]
Styles are copied onto the hidden textarea and computed there in a more isolated environment to determine how much height should be applied onto your textarea. This is done with the help of `getComputedStyle` - unfortunately it seems that it's broken in IE for this case (as well as for some others).

It seems to me that it breaks because of some combination of tag & descendant selectors (ur 1st and 3rd ones). Not sure if we can fix it on our side (I don't have idea how to do it in a non-obscure way), so I'd advise not using `inherit` for this case.

**Comment by cecton** [comment]
Yes that's how I fixed it for now. I think it was important that there is a ticket referencing that issue somewhere even though it's a pretty edge use case.

To circumvent this issue:

I normally have a CSS per component and I was using the inherit in order to be able to set the font-size in the CSS of the major component that is using my minor component (which is using react-textarea-autosize). In the major component component's CSS I made a selector that goes to this textarea only and I set the font-size there.

I think we can safely close this ticket. Maybe mentioning it in the known issues would be a good idea.
